zilla crashes with `java.lang.NoClassDefFoundError: com/google/gson/JsonElement`

<!--
  - Please search to see if an issue already exists for the issue you encountered. Duplicate issues will be closed. [Current GitHub Issues](https://github.com/aklivity/zilla/issues)
  - Please make sure your Zilla version is up to date. [Latest Zilla Release](https://github.com/aklivity/zilla/releases/latest)
-->

**Describe the bug**
When running the [http.kafka.proto.json](https://github.com/aklivity/zilla-examples/tree/example-startup-compose/http.kafka.proto.json) example and sending a request, the process fails with a `NoClassDefFoundError for com.google.gson.JsonElement`. 

**To Reproduce**
Steps to reproduce the behavior:
1. Go to [http.kafka.proto.json example](https://github.com/aklivity/zilla-examples/tree/example-startup-compose/http.kafka.proto.json)
2. Start the example using ./setup.sh
3. Trigger mentioned request:
```
curl 'http://localhost:7114/requests' \
--header 'Content-Type: application/json' \
--data '{
    "message": "hello world",
    "count": 10
}' -v
```
4. See error

```
org.agrona.concurrent.AgentTerminationException: java.lang.NoClassDefFoundError: com/google/gson/JsonElement
	at io.aklivity.zilla.runtime.engine@0.9.125/io.aklivity.zilla.runtime.engine.internal.registry.EngineWorker.doWork(EngineWorker.java:858)
	at org.agrona.core/org.agrona.concurrent.AgentRunner.doWork(AgentRunner.java:304)
	at org.agrona.core/org.agrona.concurrent.AgentRunner.workLoop(AgentRunner.java:296)
	at org.agrona.core/org.agrona.concurrent.AgentRunner.run(AgentRunner.java:162)
	at java.base/java.lang.Thread.run(Thread.java:1570)
Caused by: java.lang.NoClassDefFoundError: com/google/gson/JsonElement
	at io.aklivity.zilla.runtime.model.protobuf@0.9.125/io.aklivity.zilla.runtime.model.protobuf.internal.protobuf.util.JsonFormat$Parser.merge(JsonFormat.java:485)
	at io.aklivity.zilla.runtime.model.protobuf@0.9.125/io.aklivity.zilla.runtime.model.protobuf.internal.ProtobufWriteConverterHandler.serializeJsonRecord(ProtobufWriteConverterHandler.java:179)
	at io.aklivity.zilla.runtime.engine@0.9.125/io.aklivity.zilla.runtime.engine.catalog.CatalogHandler.encode(CatalogHandler.java:115)
	at io.aklivity.zilla.runtime.model.protobuf@0.9.125/io.aklivity.zilla.runtime.model.protobuf.internal.ProtobufWriteConverterHandler.convert(ProtobufWriteConverterHandler.java:81)
	at io.aklivity.zilla.runtime.binding.kafka@0.9.125/io.aklivity.zilla.runtime.binding.kafka.internal.cache.KafkaCachePartition.writeProduceEntryContinue(KafkaCachePartition.java:901)
	at io.aklivity.zilla.runtime.binding.kafka@0.9.125/io.aklivity.zilla.runtime.binding.kafka.internal.stream.KafkaCacheClientProduceFactory$KafkaCacheClientProduceFan.onClientInitialData(KafkaCacheClientProduceFactory.java:733)
	at io.aklivity.zilla.runtime.binding.kafka@0.9.125/io.aklivity.zilla.runtime.binding.kafka.internal.stream.KafkaCacheClientProduceFactory$KafkaCacheClientProduceStream.onClientInitialData(KafkaCacheClientProduceFactory.java:1354)
	at io.aklivity.zilla.runtime.binding.kafka@0.9.125/io.aklivity.zilla.runtime.binding.kafka.internal.stream.KafkaCacheClientProduceFactory$KafkaCacheClientProduceStream.onClientMessage(KafkaCacheClientProduceFactory.java:1276)
	at io.aklivity.zilla.runtime.engine@0.9.125/io.aklivity.zilla.runtime.engine.internal.registry.EngineWorker.handleReadInitial(EngineWorker.java:1277)
	at io.aklivity.zilla.runtime.engine@0.9.125/io.aklivity.zilla.runtime.engine.internal.registry.EngineWorker.handleRead(EngineWorker.java:1244)
	at io.aklivity.zilla.runtime.engine@0.9.125/io.aklivity.zilla.runtime.engine.internal.concurent.ManyToOneRingBuffer.read(ManyToOneRingBuffer.java:229)
	at io.aklivity.zilla.runtime.engine@0.9.125/io.aklivity.zilla.runtime.engine.internal.registry.EngineWorker.doWork(EngineWorker.java:852)
	... 4 more
	Suppressed: java.lang.Exception: [engine/data#3]	[0x030300000000002d] streams=[consumeAt=0x00005a10 (0x0000000000005a10), produceAt=0x00005a70 (0x0000000000005a70)]
		at io.aklivity.zilla.runtime.engine@0.9.125/io.aklivity.zilla.runtime.engine.internal.registry.EngineWorker.doWork(EngineWorker.java:856)
		... 4 more
Caused by: java.lang.ClassNotFoundException: com.google.gson.JsonElement
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	... 16 more
stopped
```

**Expected behavior**
Validation should work as expected & the request should be processed.

--- a/runtime/model-protobuf/pom.xml
+++ b/runtime/model-protobuf/pom.xml
@@ -26,6 +26,7 @@
   <properties>
     <jacoco.coverage.ratio>0.89</jacoco.coverage.ratio>
     <jacoco.missed.count>0</jacoco.missed.count>
+    <protobuf.version>3.25.5</protobuf.version>
   </properties>
 
   <dependencies>
@@ -44,12 +45,12 @@
     <dependency>
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
-      <version>3.25.5</version>
+      <version>${protobuf.version}</version>
     </dependency>
     <dependency>
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java-util</artifactId>
-      <version>3.24.4</version>
+      <version>${protobuf.version}</version>
     </dependency>
     <dependency>
       <groupId>org.antlr</groupId>
@@ -170,6 +171,7 @@
             <configuration>
               <artifactSet>
                 <includes>
+                  <include>com.google.code.gson:gson</include>
                   <include>com.google.protobuf:protobuf-java</include>
                   <include>com.google.protobuf:protobuf-java-util</include>
                 </includes>
@@ -178,6 +180,10 @@
                 <relocation>
                   <pattern>com.google.protobuf</pattern>
                   <shadedPattern>io.aklivity.zilla.runtime.model.protobuf.internal.protobuf</shadedPattern>
+                </relocation>
+                <relocation>
+                  <pattern>com.google.gson</pattern>
+                  <shadedPattern>io.aklivity.zilla.runtime.model.protobuf.internal.gson</shadedPattern>
                 </relocation>
               </relocations>
               <keepDependenciesWithProvidedScope>true</keepDependenciesWithProvidedScope>

